### PR TITLE
Add may 2023 changelog

### PR DESCRIPTION
**Pull request by matifali**

### CODE REVIEW

File: changelogs/2023-05-30/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: What is New in May 2023
+date: May 30, 2023
+authors:
+  - { name: Muhammad Atif Ali, avatar: https://github.com/matifali.png }
+  - { name: Ben Potter, avatar: https://github.com/bpmct.png }
```
Comment (bpmct) [inline changelogs/2023-05-30/index.md:6]: You don't have to put me as an author here, this was all you :)

File: changelogs/2023-05-30/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+![banner](.\static\banner.png)
+
+In the last few months, we have been working on exciting new features for Coder. We are excited to share them with you today.
+
+## Workspace Proxies
+
+Workspace proxies provide low-latency experiences for geo-distributed teams.
+
+Coder's networking makes the best effort to connect directly to a workspace. In situations where this is not possible, such as dashboard connections, workspace proxies can reduce the distance the network traffic needs to travel.
```
[the remark this answers is not in the document]
Comment (bpmct) [reply changelogs/2023-05-30/index.md:18]: I guess we need to change this in the docs too. Done: https://github.com/coder/coder/pull/7735

File: changelogs/2023-05-30/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+[Envbox](https://github.com/coder/envbox) is an image developed and maintained by Coder that bundles the sysbox runtime. It works by starting an outer container that manages the various sysbox daemons and spawns an unprivileged inner container that acts as the user's workspace. The inner container is able to run system-level software similar to a regular virtual machine (e.g., systemd, dockerd, etc.). Envbox offers the following benefits over running sysbox directly on the nodes:
+
+- No custom runtime installation or management on your Kubernetes nodes.
+- No limit to the number of pods that run Envbox.
+
+Refer to the [docs](https://coder.com/docs/v2/latest/templates/docker-in-workspaces#envbox) for more information.
+
+## Template Editor
+
+The template editor is a new feature that allows you to edit and duplicate templates directly from the Coder dashboard. This feature is available to all users with the `template-admin` or `owner` role.
```
Comment (bpmct) [inline changelogs/2023-05-30/index.md:37]:
```suggestion
Edit & duplicate templates directly from the Coder dashboard. This feature is available to all users with the `template-admin` or `owner` role.
```

File: changelogs/2023-05-30/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+![template editor](.\static\template-editor.gif)
+
+## Fly.io Starter Template
+
+We have added a new starter template for [Fly.io](https://fly.io/). Fly.io is a platform for running applications globally. It is a great fit for Coder workspaces because it allows you to run them close to your developers. See the blog post [here](https://coder.com/blog/remote-developer-environments-on-fly-io) for more information.
```
Comment (bpmct) [inline changelogs/2023-05-30/index.md:43]: Thoughts on also showing a screenshot with all the regions?

File: changelogs/2023-05-30/index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+![Fly.io Starter Template](.\static\fly-io-starter-template.png)
+
+### Other Exciting Features
+
+- Ability to use non-localhost addresses with `coder port-forward` [docs](https://coder.com/docs/v2/latest/cli/port-forward)
+- Generate _Open In Coder_ buttons for your templates to provide a one-click experience for your developers. [#7501](https://github.com/coder/coder/pull/7501)
```
Comment (bpmct) [inline changelogs/2023-05-30/index.md:49]: Hmm.. I think this could actually deserve its own section with the embed page. It is a significant UI change :)

**Review by bpmct — APPROVED** [review]
Awesome post! Only minor comments.

Also tagging @ammario if he wants to take a look :)

File: changelogs/2023-05-30/static/banner.png
Comment (bpmct) [inline changelogs/2023-05-30/static/banner.png:1]: We have been using DALL-E to generate the changelog pictures. I'd stick with the theme of using those "abstract" pictures. We could work to make a more formal changelog banner down the road.

Can you generate one that compliments our site theme and then crop it to the proper size?

Screenshot 2023-05-31 at 9 58 02 AM
Screenshot 2023-05-31 at 9 57 46 AM
Screenshot 2023-05-31 at 9 57 55 AM

File: changelogs/2023-05-30/index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+## Workspace Proxies
+
+Workspace proxies provide low-latency experiences for geo-distributed teams.
+
+Coder's networking makes the best effort to connect directly to a workspace. In situations where this is not possible, such as dashboard connections, workspace proxies can reduce the distance the network traffic needs to travel.
+
+A workspace proxy is a relay connection a developer can use when connecting with their workspace over ssh, a workspace app, port forwarding, etc. Coder automatically selects the best proxy for the developer based on their location and the location of the workspace.
+
+For deployment instructions, see [Docs](https://coder.com/docs/v2/latest/admin/workspace-proxies).
```
[the remark this answers is not in the document]
Comment (matifali) [reply changelogs/2023-05-30/index.md:22]: better to mention. As it needs to be enabled explicitly

File: changelogs/2023-05-30/index.md (hunk 5 above)
Comment (matifali) [reply changelogs/2023-05-30/index.md:49]: the embed button?

**Review by ammario — COMMENTED** [review]
Sick, thanks @matifali!

**Pull request merged**